tests/entanglement_management/test_purification.py

```diff
@@ -598,23 +602,20 @@ def test_BBPSSW_psi_minus_psi_minus():
 
         if ep1.meas_res == 0:
             counter += 1
-            assert complex_array_equal(phi_plus, state)
+            assert complex_array_equal(psi_plus, state)
```

Comment:
This was the bug, I take it?

Reply:
That was one of them. The other is the actual `def complex_array_equal`.